### PR TITLE
refactor Input events. Use EventsManager on ReactEcs

### PR DESCRIPTION
**Pull request by gonpombo8**
# What ?? 
Input.isPressed for global input state
Input.isTriggered for specific entity action events
Mark Input.isClicked as internal for the moment
Mark EventsManager.click as internal
Remove React UI Events internal Map, and use EventsManager.onPointerDown instead (Lets the engine handle all this logic)
Add 100% Coverage, and fail if it is not covered 🎉

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -20,7 +26,7 @@ module.exports = {
   coveragePathIgnorePatterns: [
     "scripts",
     "test",
-    "packages/@dcl/react-ecs"
+    // "packages/@dcl/react-ecs"
```
Comment (leanmendoza) [inline jest.config.js:29]: comment

File: packages/@dcl/ecs/src/components/index.ts [hunk 2]
```diff
@@ -40,6 +40,6 @@ export function defineSdkComponents(engine: PreEngine) {
  * })
  * ```
  */
-export function makeCollisionMask(...layers: ColliderLayer[]): number {
-  return layers.reduce((item, currentValue) => item | currentValue)
-}
+// export function makeCollisionMask(...layers: ColliderLayer[]): number {
```
Comment (leanmendoza) [inline packages/@dcl/ecs/src/components/index.ts:43]: comment

File: packages/@dcl/ecs/src/index.ts [hunk 3]
```diff
@@ -14,6 +14,6 @@ export * from './runtime/types'
 
 export * from './runtime/observables'
 export * from './runtime/temp-fp/Observable'
-export * from './runtime/MessageBus'
+export * from './runtime/messageBus'
 
-export { makeCollisionMask } from './components'
+// export { makeCollisionMask } from './components'
```
Comment (leanmendoza) [inline packages/@dcl/ecs/src/index.ts:19]: comment

**Comment by nearnshaw** [comment]
@gonpombo8 
The name of `Input.isActive` can potentially be confused with `Input.IsActionDown`

The first triggers only on the tick right after the button went down. It's an event that is already over.
The second triggers also on every tick that the button remains down. It's an ongoing event.

How about `Input.wasActive` or `Input.wasActivated`
or `Input.wasTriggered`?

**Review by leanmendoza — APPROVED** [review]
LGTM 😄 I marked the dead code to remove it

**Comment by leanmendoza** [comment]
> @gonpombo8 The name of `Input.isActive` can potentially be confused with `Input.IsActionDown`
> 
> The first triggers only on the tick right after the button went down. It's an event that is already over. The second triggers also on every tick that the button remains down. It's an ongoing event.
> 
> How about `Input.wasActive` or `Input.wasActivated` or `Input.wasTriggered`?

I'm more towards keeping the `Input.isActive`. The event is known in the current tick, so you ask for something that is happening and known in the current tick.

About `Input.isActionDown`, it remains in continuity. So it's valid to say that every tick is true the event is happening. We can orientate it to `Input.isActionPressing` or something that remarks continuity.

**Pull request merged**